pkg/gen/input/workflows/internal/file/create_release_branch.go

```diff
@@ -0,0 +1,136 @@
+package file
+
+import (
+	"github.com/giantswarm/devctl/pkg/gen/input"
+	"github.com/giantswarm/devctl/pkg/gen/input/workflows/internal/params"
+)
+
+func NewCreateReleaseBranchInput(p params.Params) input.Input {
+	i := input.Input{
+		Path:         params.RegenerableFileName(p, "create_release_branch.yaml"),
+		TemplateBody: createReleaseBranchTemplate,
+	}
+
+	return i
+}
+
+var createReleaseBranchTemplate = `# DO NOT EDIT. Generated with:
+#
+#    devctl gen workflows
+#
+
+# Creates a branch for the 'previous' minor version when a 'new' minor version is tagged
+
+name: create-minor-version-branch
+
+on:
+  push:
+    tags: ['v*.*.*']
```

Comment:
One more thing to check is if this workflow will be run at all if the tag is created by another workflow. Apparently GH workflows are not triggered by events from other workflows. See https://github.com/giantswarm/kubectl-gs/pull/64

Reply:
I'm pretty sure it won't be triggered when tag is created by another workflow https://docs.github.com/en/actions/reference/events-that-trigger-workflows#triggering-new-workflows-using-a-personal-access-token

So you'd need to glue that job to `Create Release` workflow I guess.

Reply:
Yeah you're right, so to have any separate workflows after `Create Release` we'd have to make it use an OAuth token instead. Do you have a problem gluing this to Create Release?

Reply:
> Do you have a problem gluing this to Create Release?

Not at all.

Reply:
Added this to create_release